.dockerignore

```diff
@@ -1 +1,2 @@
-test/output
+.github/
+tests/
```

Comment:
The tests are passed to the test scripts via a volume mount, but they don't have to be present when building the Dockerfile. Having them here has the benefit of being able to change the tests without having to rebuild the Docker image.